src/exts/garlic.py

```diff
@@ -22,10 +22,13 @@ async def on_message(self, message: Message) -> None:
             await message.add_reaction(GARLIC)
 
     @slash_command(name="garlic", description="Get your garlic count.")
-    async def garlic(self, itr: CommandInteraction) -> None:
-        count = await self.bot.manager.get_user_garlic(itr.author)
+    async def garlic(self, itr: CommandInteraction, user: User) -> None:
+        if user is None:
+            count = await self.bot.manager.get_user_garlic(itr.author)
+        else:
+            count = await self.bot.manager.get_user_garlic(user)
 
-        await itr.send(f"{GARLIC} You have {count} garlic {GARLIC}")
+        await itr.reply(f"{GARLIC} {'You' if user is None else str(user)} {'have' if user is None else 'has'} {count} garlic {GARLIC}")
```

Comment:
I feel like just having the send inside the if statements makes more sense than using multiple inline ternary expressions.